test/unittests/test_loader.cpp

```diff
@@ -301,6 +301,11 @@ TEST_F(loader, load_and_create_failure)
     EXPECT_TRUE(vm == nullptr);
     EXPECT_EQ(ec, EVMC_LOADER_INSTANCE_CREATION_FAILURE);
     EXPECT_STREQ(evmc_last_error_msg(), "creating EVMC instance of failure.vm has failed");
+    EXPECT_TRUE(evmc_last_error_msg() == nullptr);
```

Comment:
could be `EXPECT_EQ`

Reply:
Because of the GTest issues, this would not compile in VS 2017.